### PR TITLE
[ai deploy] - new img seg app

### PR DESCRIPTION
**Pull request by MathieuBsqt**
Here is a new documentation that is linked to a new tutorial about creating an image segmentation app, deployed with OVHcloud AI Deploy tool. 

I have create all language duplicas & updated the index file.

This PR will be in draft until the image segmentation notebook's doc link is published. I need to add it in these new docs.

### CODE REVIEW

**Comment by MathieuBsqt** [comment]
I have added the notebook's documentation link to this app documentation. I have also added the code repositorys. 

I have deleted all language duplicates, as seen with Yoann. You will then need to create them. 

Here are the information about French versions : 

- **title:** AI Deploy - Tutoriel - Déployer une app de segmentation de tumeurs
- **excerpt:** Comment créer et déployer une application permettant de segmenter des tumeurs cérébrales présentes dans des images IRM
- **section:** AI Deploy - Tutoriels

File: pages/platform/ai/deploy_tuto_14_img_segmentation_app/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+> Make sure to upload the 2 files obtained during the training of your model.
+> Weight files names can change for you. Make sure that the variable `best_weights_path` (of the `variables.py` file) indicates the path of your model's weights. Change it if necessary.
+>
+
+Once your data is uploaded, it can be accessed from all OVHcloud AI tools, either with read-only (RO) or read-write (RW) permissions.
+
+> [!warning]
+>
+> Before continuing, make sure to modify the `samples_test` list in the `variables.py` file.
+> Indeed, this list must contain the patients of the test set, which were not used for the training of the model. This list will not be the same for you, since the patients were randomly shuffled. It is therefore important that you replace it with your own test samples list!
```
Comment (OVH-Team-Guides) [inline pages/platform/ai/deploy_tuto_14_img_segmentation_app/guide.en-gb.md:179]: Hello @MathieuBsqt, is "patients" the actual term you wish to use here?

Comment (MathieuBsqt) [reply pages/platform/ai/deploy_tuto_14_img_segmentation_app/guide.en-gb.md:179]: Yes, `patients` is the word I wanted to use. I agree that it is a bit confusing. In fact, we often talk about `samples` in A.I., but in this use case, the `samples` are `patients`. That's why the 2 terms are exchanged sometimes.

**Comment by MathieuBsqt** [comment]
Thanks a lot for Proofreading. That's perfect. 

I just have a question about a link that you changed. New link seems incorrect to me.

Original sentence:
refer to the following [documentation] (https://help.ovhcloud.com/csm/en-gb-public-cloud-ai-notebooks-tuto-image-segmentation-unet-tumors?id=kb_article_view&sysparm_article=KB0057606).

Modified version: 
refer to the following [documentation] (/pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors).

**Comment by OVH-Team-Guides** [comment]
> Thanks a lot for Proofreading. That's perfect.
> 
> I just have a question about a link that you changed. New link seems incorrect to me.
> 
> Original sentence: refer to the following [documentation] (https://help.ovhcloud.com/csm/en-gb-public-cloud-ai-notebooks-tuto-image-segmentation-unet-tumors?id=kb_article_view&sysparm_article=KB0057606).
> 
> Modified version: refer to the following [documentation] (/pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors).

That's the new link norm for help.ovhcloud.com, this will simplify link edition a lot. I will tell you more about in in private.

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @MathieuBsqt

**Pull request merged**